src/status_im/ui/screens/browser/events.cljs

```diff
@@ -120,6 +121,12 @@
    (when (< history-index (dec (count (:history browser))))
      (nav-update-browser cofx browser (inc history-index)))))
 
+(handlers/register-handler-fx
+ :send-qr-code-callback
```

Comment:
Please follow the guidelines for new events https://github.com/status-im/status-react/blob/develop/doc/codebase-structure-and-guidelines.md#events